mantidproject/mantid
# Add selectable Periodic Table

**Description of work.**
Selectable Periodic Table widget with event registration and unit testing - added to mantid interfaces

**Report to:** Adrian

**To test:**
Open Interfaces->Muon->ElementalAnalysis
Selecting/Deselecting elements should output "Element Clicked: Symbol" and "Table Changed: List of selected symbols" to the log

Fixes #22861 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
